kv/kv.go

```diff
@@ -85,6 +102,16 @@ type Transaction interface {
 	LockKeys(keys ...Key) error
 }
 
+// MvccSnapshot is used to get/seek a specific verion in a snaphot.
+type MvccSnapshot interface {
+	// MvccGet returns the specific version of given key, if the version doesn't
+	// exists, returns the nearest(lower) version's data.
+	MvccGet(k Key, ver Version) ([]byte, error)
```

Comment:
ignore above, I find that dbSnapshot must implement MvccSnapshot and it has Get already.